packages/react-instantsearch-core/src/core/translatable.js

```diff
@@ -23,7 +23,7 @@ export default function translatable(defaultTranslations) {
         const { translations } = this.props;
 
         const translation =
-          translations && has(translations, key)
+          translations && translations[key] !== undefined
```

Comment:
this is what potentially is a breaking change, let's decide if it's important enough to not break / leave as is

Reply:
If I understand correctly, the previous version considered `undefined` as a value (because the key exists), and therefore displayed "undefined", right? IMO this wasn't expected, then it's not a breaking change.

If the behavior changed, we should add a test to keep track of it.

Reply:
That’s what I thought too, should cover it with a test to be sure

Reply:
decided not to have this behaviour in ab81d0b